src/main/java/org/mockito/internal/stubbing/defaultanswers/ReturnsDeepStubs.java

```diff
@@ -76,8 +77,16 @@ public Object answer(InvocationOnMock invocation) throws Throwable {
         if (rawType.equals(Object.class) && !returnTypeGenericMetadata.hasRawExtraInterfaces()) {
             return null;
         }
-
-        return deepStub(invocation, returnTypeGenericMetadata);
+        try {
+            return deepStub(invocation, returnTypeGenericMetadata);
+        } catch (MockitoCantMockEnumException ex) {
+            // If the enum is not mockable, e.g. due to sealed, return the first literal as value
```

Comment:
Rather than swallowing the exception here, I would prefer to always throw the exception and let the user to decide what to return. E.g. you can use `DEEP_STUBS` as default answer and then override this one stub to specify the enum you expect to return. I don't think defaulting to the first value is expected behavior from users.